.gitignore

```diff
@@ -11,3 +11,12 @@ build/
 
 ## screeps.com account.email and account.password
 account.screeps.com.js
+
+#Ignore thumbnails created by Windows
+Thumbs.db
+
+#Ignore files and folders built by Visual Studio
+*.sln
+*.suo
+*.log
+.vs/
```

Comment:
Please use your global gitignore for that. http://stackoverflow.com/questions/7335420/global-git-ignore